docs/manual/docs/cmd/graph/teams/teams-channel-list.md

```diff
@@ -0,0 +1,36 @@
+#graph teams channel list
+
+List the channels in a specified Microsoft Teams team
+
+## Usage
+```sh
+ graph teams channel list [options]
+```
+
+## Options
+
+Option|Description
+------|-----------
+   `--help`| output usage information
+   ` -i, --groupId <groupId> `| The ID of the group to list the channels
+   ` -o, --output [output] `| Output type. json|text. Default text
+   `  --verbose `| Runs command with verbose logging
+   `  --debug `|  Runs command with debug logging
+   
+     !!! important
+   Before using this command, log in to the Microsoft Graph, using the [graph login](../login.md) command
+          
+## Remarks
+
+    !!! Attention
```

Comment:
This line shouldn't be indented. Also `attention` should be spelled with lowercase `a`